seo/ville/sorl.py

```diff
@@ -1,9 +1,32 @@
-from os import path
+import contextlib
+
+from os import path, remove
 from sorl.thumbnail.base import ThumbnailBackend, EXTENSIONS
 from sorl.thumbnail.conf import settings
 from sorl.thumbnail import default
+from tempfile import NamedTemporaryFile
 from .slugger import SEOStorage
 
+from PIL import Image, ImageColor
+
+
+@contextlib.contextmanager
+def create_copy_with_matte(source_image, matte=settings.THUMBNAIL_PADDING_COLOR):
```

Comment:
Wat is nu de default dan en wat gebeurt er als je dat niet invult

Reply:
```
from sorl.thumbnail.conf import settings
```